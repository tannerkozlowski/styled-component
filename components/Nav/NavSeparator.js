import styled from 'styled-components'

import rem from '../../utils/rem'

const NavSeparator = styled.span`
  width: ${rem(5)};
  height: ${rem(5)};
  margin: 0 ${rem(15)};

  border-radius: 50%;
  background: rgba(255, 255, 255, 0.35);
`

export default NavSeparator
