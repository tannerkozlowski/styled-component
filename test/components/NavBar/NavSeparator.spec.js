import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import NavSeparator from '../../../components/Nav/NavSeparator'

test('NavSeparator renders correctly', () => {
  const tree = renderer.create(<NavSeparator />).toJSON()

  expect(tree).toMatchSnapshot()
})
