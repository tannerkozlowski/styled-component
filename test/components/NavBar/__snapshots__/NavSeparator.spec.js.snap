// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavSeparator renders correctly 1`] = `
<span
  className="lnegGh"
/>
`;
